tests/unit/fixtures/configs/for_build_tfds_classification.py

```diff
@@ -33,17 +33,16 @@
 )
 
 
-class ClassificationDataset(ClassificationBase):
-    extend_dir = "custom_delta_mark_classification/for_train"
-    validation_extend_dir = "custom_delta_mark_classification/for_validation"
+class ClassificationDataset(ImageFolderBase):
+    extend_dir = "dummy_classification"
 
 
 IS_DEBUG = False
 
 NETWORK_CLASS = LmnetV0Quantize
 DATASET_CLASS = ClassificationDataset
 
-IMAGE_SIZE = [128, 128]
+IMAGE_SIZE = [28, 28]
```

Comment:
same as above

Reply:
This is also dataset dependent parameter. 
(28, 28) is the image size of delta-mark dataset.